src/components/tools/NewToolRequestDialog.js

```diff
@@ -29,7 +29,7 @@ export default function NewToolRequestDialog(props) {
     const [requestError, setRequestError] = useState();
 
     const { t } = useTranslation("tools");
-    const { t: i18nUtil } = useTranslation("utils");
```

Comment:
This was causing `404`. There is no corresponding `utils.json` file. There is only `util.json`